src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
```rbd image-meta set``` already sends out a update notification. The only missing piece is to have ```image/RefreshRequest``` query the metadata.